x/superfluid/keeper/edge_case_test.go

```diff
@@ -142,3 +146,85 @@ func (s *KeeperTestSuite) TestTryUnbondingSuperfluidLockupDirectly() {
 		})
 	}
 }
+
+func (s *KeeperTestSuite) handleEquivocationEvidence(ctx context.Context, evidence *types.Equivocation) error {
```

Comment:
Yes exactly, this is now a private method in the sdk, but its kind of silly given all the methods within it are public. So instead of staying further from the sdk, and because this is only used in tests, I added here manually.